pyaerocom/data/variables.ini

```diff
@@ -5283,3 +5283,17 @@ minimum = -150
 maximum = 150
 dimensions = time,lev,station
 comments_and_purpose = For physics evaluation and chemistry interpretation above stations' locations.
+
+[ratpm10pm25]
+description = ratio of pm10 and pm25
+unit = 1
+minimum = -150
+maximum = 150
+dimensions = time,lat,lon
+
+[ratpm25pm10]
+description = ratio of pm25 and pm10
+unit = 1
+minimum = -150
+maximum = 150
+dimensions = time,lat,lon
```

Comment:
please add an `\n` ad the end of the file

Reply:
done